javascript/monaco-editor/security/audit/monaco-hover-htmlsupport.yaml

```diff
@@ -29,5 +29,5 @@ rules:
     message: >-
       If user input reaches `HoverProvider` while `supportHml` is set to `true` it may introduce an XSS vulnerability.
       Do not produce HTML for hovers with dynamically generated input.
-    languages: [typescript, javascript]
+    langtspescript, js]
```

Comment:
This is WEIRD - see `node-expat` changes for example of exact same replace that worked fine.